### ISSUE TITLE
Forms/AJAX Loading Style

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#20]
**Requirement**: We should not automatically refresh entire pages.

**Why is this required?**
- When a page auto-refreshes, screen readers will begin to read the content of the page again. This can be very frustrating for screen reader users who may find it difficult to get to the end of the page before it auto-refreshes again
- If content (e.g. a 'breaking news' box) needs to update dynamically, this should be implemented by refreshing only that content instead of the entire page. (ARIA guidelines for accessibility in dynamic web pages may apply here.) 

Provide an "update now" button instead of automatically updating the content.
